### DISCUSSION TITLE
🚀 Start UI [web] is now using React Router v6

### DISCUSSION THREAD

**Discussion by yoannfleurydev** [BearStudio/start-ui-web#140] [Announcements]
🚀 Start UI [web] is now using [React Router v6](https://reactrouter.com/docs/en/v6), the latest version of React Router that is a mix of the previous version and its sibling project Reach Router.

Thanks for following and using this project. Feel free to share [this tweet](https://twitter.com/startui_/status/1475814504634232835?s=20) or this discussion on your social networks, it helps us a lot.

<img width="1280" alt="Thumbnail GitHub Start UI  web" src="https://user-images.githubusercontent.com/3920615/147908099-58b12d60-27c3-4e97-a339-6c13fedb2889.png">

**Comment by winner106** [comment]
`Start UI [web]` is absolutely a wonderful project!

**Reply by yoannfleurydev** [reply]
Thanks a lot, this kind of feedback is a motive for the team to invest on it!
